todo_list/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from todo_list.models import Task, Tag
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports) 